mungegithub/mungers/needs_rebase.go

```diff
@@ -60,6 +60,12 @@ func (NeedsRebaseMunger) Munge(obj *github.MungeObject) {
 		obj.RemoveLabel(needsRebase)
 	}
 	if !mergeable && !obj.HasLabel(needsRebase) {
+		glog.Infof("PR: %d needs rebase", *obj.Issue.Number)
```

Comment:
I'd rather not spam the logs with this, since as the person looking at the logs for problems, this doesn't tell me anything.